src/apps/AdminApp/AdminRoot.js

```diff
@@ -507,8 +513,7 @@ class AdminRoot extends Component {
         const toAck = [feed_user.id];
 
         if(command_type === "audio-out") {
-            gdm.send(cmd, toAck, (cmd) => gateway.sendCmdMessage(cmd)).
-            then(() => {
+            gdm.send(cmd, toAck, (cmd) => gateway.sendCmdMessage(cmd)).then(() => {
```

Comment:
code style: each promise chain (`then`, `catch` or `finally`) should be on its own line.
Please fix throughout this PR